### PR TITLE
Removed packages-panel style

### LINKED ISSUE #2709: "Copyright" Footer Covers Installed Packages And All Other Content
If user narrows the browser window vertically, the "Copyright" footer covers the packages list (and all other lists as well). 

**Testflow:**
1. Login to Kura as admin/admin
2. Go to Packages, install several packages
3. Narrow the browser window vertically (see screenshot bellow)
4. Observe the "Copyright" footer that covers packages list 

**Expected behavior**
Footer should never cover anything regardles of browser size. 

**Screenshot:**
Screenshot 2020-03-16 at 11 44 02

Target Environment (please complete the following information):
Board: Raspberry Pi 3
OS version: Linux raspberrypi 4.19.75-v7+ # 1270 SMP Tue Sep 24 18:45:11 BST 2019 armv7l GNU/Linux
Additional info: /

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR removes the packages-panel style that causes the "Copyright" Footer to interfere with other contents of the page.

**Related Issue:** This PR fixes/closes #2709 

Signed-off-by: pierantoniomerlino <pierantonio.merlino@eurotech.com>

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
Hey @pierantoniomerlino,
I have checked this PR and it looks ok, it successfully solves issue 2709 so it can be merged.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.ui.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     </ui:style>
 
 
-    <b:Container fluid="true" addStyleNames="packages-panel">
+    <b:Container fluid="true">
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/packages/PackagesPanelUi.ui.xml:40]: Please change the header to: "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"

File: kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/css/denali.css [hunk 2]
```diff
@@ -1041,13 +1041,6 @@ Packages panel dropzone START
     color:  white;
 }
 
-.packages-panel {
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/css/denali.css:1044]: Please change header to: "* Copyright (c) 2011, 2020 Eurotech and/or its affiliates and others"

**Review by LeoNerdoG — COMMENTED** [review]
Hey @pierantoniomerlino,
can you please change the header in the files commented bellow? The year of last change is not set to "2020".

**Review by LeoNerdoG — APPROVED** [review]

**Review by MMaiero — APPROVED** [review]

**Pull request merged**